Source/control.cpp

```diff
@@ -2290,7 +2290,7 @@ void CheckChrBtns()
 			    && MouseX <= attribute_inc_rects[i][0] + attribute_inc_rects[i][2]
 			    && MouseY >= attribute_inc_rects[i][1]
 			    && MouseY <= attribute_inc_rects[i][3] + attribute_inc_rects[i][1]) {

```

Comment:
We should consider making `attribute_inc_rects` a struct with fields:

```c
typedef struct Rect {
   int x;
   int y;
   int width;
   int height;
} Rect;
```

Then this code would be easier to read.

```c
 			    && MouseX <= attribute_inc_rects[i].x + attribute_inc_rects[i].width
 			    && MouseY >= attribute_inc_rects[i].y
 			    && MouseY <= attribute_inc_rects[i].y + attribute_inc_rects[i].height) { 
```